Please check the travis builds for lint errors, too:

```
C:195, 0: Unnecessary parens after 'if' keyword (superfluous-parens)
C:196, 0: Unnecessary parens after 'return' keyword (superfluous-parens)

./homeassistant/components/light/flux_led.py:192:1: D202 No blank lines allowed after function docstring
```

--- a/homeassistant/components/light/flux_led.py
+++ b/homeassistant/components/light/flux_led.py
@@ -191,7 +191,6 @@ class FluxLight(Light):
     @property
     def supported_features(self):
         """Flag supported features."""
-
         if self._mode is MODE_RGBW:
             return SUPPORT_FLUX_LED | SUPPORT_WHITE_VALUE
 

--- a/homeassistant/components/light/flux_led.py
+++ b/homeassistant/components/light/flux_led.py
@@ -192,8 +192,8 @@ class FluxLight(Light):
     def supported_features(self):
         """Flag supported features."""
 
-        if(self._mode is MODE_RGBW):
-            return (SUPPORT_FLUX_LED | SUPPORT_WHITE_VALUE)
+        if self._mode is MODE_RGBW:
+            return SUPPORT_FLUX_LED | SUPPORT_WHITE_VALUE
 
         return SUPPORT_FLUX_LED
 

